### DISCUSSION TITLE
What integration examples would you like to see?

### DISCUSSION THREAD

**Discussion by peelar** [saleor/saleor#10434] [Ideas]
Hello 👋 

We want Saleor to be headquarters for your e-commerce business, and that requires integrating it neatly with other tools. What integration examples could you use the most right now?

We are making moves to improve our content department, so the best proposals from this discussion will get example apps with a description 🎉 .

**Comment by marianmazarovici** [comment]
Integration with Salesforce with be a lifesaver for us at VONQ.com
Our fulfillment is done through Saleforce
Saleor is the source of truth for products, orders, contracts (gift cards) and discounts and needs to get synced to Salesforce on update.
We were planning on doing it internally but happy to partner up
